docs/api-reference/carto/carto-sql-layer.md

```diff
@@ -70,14 +70,14 @@ new CartoSQLLayer({});
 To use pre-bundled scripts:
 
 ```html
-<script src="https://unpkg.com/deck.gl@^8.5.0/dist.min.js"></script>
-<script src="https://unpkg.com/@deck.gl/carto@^8.5.0/dist.min.js"></script>
+<script src="https://unpkg.com/deck.gl@^8.7.0/dist.min.js"></script>
```

Comment:
Same here, we should remove this layer in 8.7

Reply:
Split out into separate PR: https://github.com/visgl/deck.gl/pull/6402